generators/app/templates/client/package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "<%= _.kebabCase(appName) %>",
   "dependencies": {
+    "react-hot-loader": "^3.0.0-beta.3",
+    "style-loader": "^0.13.1",
+    "babel-core": "^6.0.20",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^6.2.4",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "babel-polyfill": "^6.9.1",
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
-    "babel-preset-stage-1": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
```

Comment:
Agreed, and even `babel-plugin-transform-decorators-legacy` should be removed in that case. But we are always tied to webpack anyway because of all those kind of loaders and magic.